src/ReleasePage/releases.json

```diff
@@ -1,4 +1,29 @@
 [
+  {
+    "label": "Release 1.2.1",
+    "emoji": "dna",
+    "version": "1.2.1",
+    "date": "May 1, 2020",
+    "target": "internal",
+    "description": "This is an interim internal release containing the PASS1A and PASS1B phenotypic data.",
+    "readme_file_location": "https://docs.google.com/document/d/1JjVlHeTg1ahzR06H72kxlbP1ZMaJEsqRkswVh9JweyM",
+    "result_files": {
+      "bucket_name": "mmotrpac-internal-release1-2-results",
```

Comment:
I think this is a typo (mmotrpac -> motrpac)

Reply:
Great catch! Thanks, @ysyskim!